sandbox/sandbox.go

```diff
@@ -1,3 +1,5 @@
+// +build !windows
```

Comment:
I know it can be frustrating to see windows builds broken all the time. But this cannot be a non-windows version. I don't think we should duplicate the interface itself